catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/common/CsexraAttributeTest.java

```diff
@@ -93,6 +93,17 @@ public void testNiirsOther() throws NitfFormatException {
 
     }
 
+    @Test
+    public void testNiirsEmptyString() throws NitfFormatException {
+        when(tre.getFieldValue(CsexraAttribute.PREDICTED_NIIRS_SHORT_NAME)).thenReturn("");
+
+        Serializable actual = CsexraAttribute.PREDICTED_NIIRS_ATTRIBUTE.getAccessorFunction()
+                .apply(tre);
+
+        assertThat(actual, nullValue());
+
```

Comment:
Some of the other unit tests in this class have an extra line, but I fixed it anyway.